### PR TITLE
Fixes routing example

### LINKED ISSUE #6411: Fix example on 'Route and Navigator Refactoring' page
### Page URL

https://docs.flutter.dev/release/breaking-changes/route-navigator-refactoring

### Page source

https://github.com/flutter/website/tree/main/src/docs/release/breaking-changes/route-navigator-refactoring.md

### Describe the problem

image

Typo and other unclear documentation for migrating from `RouteSettings.isInitialRoute`

### Expected fix

Suggested changes are attached in the screenshot above.

### Additional context

I would "fix this myself" but didn't want to go through a multi-step checklist to submit a PR to fix a typo :^)

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #6411

### CODE REVIEW

**Review by chunhtai — APPROVED** [review]
LGTM

File: src/release/breaking-changes/route-navigator-refactoring.md [hunk 1]
```diff
@@ -88,16 +88,16 @@ MaterialApp(
...
-for the route name.
+One way is to set an explicit value for `MaterialApp.initialRoute`.
+You can then test for this value in place of `isInitialRoute`.
+As `initialRoute` inherits its default value outside of Flutter's scope,
+you must set an explicit value for it.
 
 ```dart
 MaterialApp(
-  initialRouteName: 'fakeSplash',
+  initialRoute: '/', // default value cannot be guaranteed at runtime
```
Comment (parlough) [inline src/release/breaking-changes/route-navigator-refactoring.md:98]: This reads as if it's a warning about the value you pass in as `initialRoute`, but it seems the intention is to explain why an explicit `initialRoute` is needed in this situation. Explain the goal of the line more explicitly. Perhaps something like: "Explicitly set the initial route, as its default value can be manipulated."?

**Review by parlough — COMMENTED** [review]
Thanks!

One comment on the comment:

**Review by parlough — APPROVED** [review]
Thanks!

**Pull request merged**